snappy_pipeline/apps/snappy_snake.py

```diff
@@ -246,7 +246,7 @@ def run(wrapper_args):
 def drmaa_available():
     """Return whether DRMAA execution is available"""
     try:
-        import drmaa
```

Comment:
This seems legit. Maybe we can rewrite this method using `importlib`?

Reply:
I've always though the try/except mechanism was the idiomatic way to do this in Python. I have no preference, though. We could adress this in #6.

Reply:
I don't think there is anything wrong with the approach, but automatic tools apparently will flag and remove it.